tests/pkcli/roles_test.py

```diff
@@ -55,5 +55,6 @@ def test_flash_list_role_by_email(auth_fc):
     pkunit.data_dir().join("db").copy(srdb.root())
     auth_fc.sr_email_login(e, sim_type="flash")
     roles.add(e, *r)
+    r.append("user")
     pkunit.pkeq(r, roles.list(e))
```

Comment:
Again this makes an incorrect assumption. The test should only make sure that there's an entry for sim_type flash.